lib/Swagger/src/Configuration.php

```diff
@@ -89,7 +89,7 @@ class Configuration
      *
      * @var string
      */
-    protected $host = 'https://localhost/user-preference';
+    protected $host = 'https://localhost/';
```

Comment:
Correct, `setHost` overrides this. It wouldn't work as-is anyway, because it's trying to access the service on the `localhost` domain.

The reason this exists is because we share a single `ApiClient` among all consumers of auto-generated code.